packages/manager/modules/pci/src/projects/project/training/jobs/info/info.controller.js

```diff
@@ -75,21 +75,21 @@ export default class PciTrainingJobsInfoController {
       duration += moment().diff(this.start) / 1000;
     }
 
+    // eslint-disable-next-line no-underscore-dangle
+    const date = moment(moment.duration(duration, 'seconds')._data);
```

Comment:
In the CLI we only display minutes if > 2m && < 2h, only hours if > 2h && < 2d, ...
It may be cleaner to have a similar display width all the time.